jodconverter-core/pom.xml

```diff
@@ -10,7 +10,7 @@
   <parent>
     <groupId>org.alfresco</groupId>
     <artifactId>alfresco-super-pom</artifactId>
-    <version>8</version>
+    <version>12</version>
```

Comment:
Needed  to be able to use mvn help:evaluate to ensure we don't already have the same version tagged in nexus.